projects/plugins/crm/includes/ZeroBSCRM.Migrations.php

```diff
@@ -17,9 +17,8 @@
   / Breaking Checks
    ====================================================== */
 
-
-
-
+require_once ABSPATH . 'wp-admin/includes/class-wp-filesystem-base.php';
+require_once ABSPATH . 'wp-admin/includes/class-wp-filesystem-direct.php';
```

Comment:
Load the files when they are going to be used. If not, we will load those files on each request. 

For example, we could add a new function to load it:

```
function load_wp_filesystem() {
    require_once ABSPATH . 'wp-admin/includes/file.php';
    WP_Filesystem();
}
```

Then we could call it forIn `zeroBSCRM_migration_560_move_custom_file_upload_box` and from `zeroBSCRM_migration_560_move_file_array` and then use the global variable `global $wp_filesystem;`